### ISSUE TITLE
Mod does not work without More Slugcats enabled

### ISSUE THREAD

**Issue by mfrancis33** [HarvieSorroway/Biography#1]
The mod does not work without the More Slugcats mod enabled. The source of this issue stems from the fact that the "COLLECTION" button only exists if MSC is enabled, which is where this mod tries to place its button.

Suggestions: either add MSC as a dependency, add a check to move the button's location if MSC isn't enabled, or move the button in general off of a mod-specific button.

**Comment by StarmantheBlaziken** [comment]
Is this why the mod gives me a black screen upon start up? I have one just here after having to fiddle with the files to get the respective Remix options working again for this...
I am getting a feel for Rainworld with Monk before I add the DLC, unless maybe I should add it anyways for the bigger experience.
![20230828133622_1](https://github.com/HarvieSorroway/Biography/assets/56205505/691d7061-a086-41fb-8b69-54c8f40fc399)

**Comment by HarvieSorroway** [comment]
yes it is because the Collection button wont exist if msc not enabled, while biography use it.

**Comment by StarmantheBlaziken** [comment]
Unless you can perhaps make it so it could work fine in some way with and without MSC which would be nice... Yeah, best to make it so that on Steam and in-menu it asks or depends on the MSC DLC mod.
